bitmovin/encoding/statistics.js

```diff
@@ -4,57 +4,52 @@ import BitmovinError from '../BitmovinError';
 import {isValidApiRequestDateString} from '../DateUtils';
 
 export const statistics = (configuration, http) => {
-  const { get } = http;
+  const {get} = http;
+
+  const addOptionsToUrl = (url, options) => {
+    let newUrl = url;
+    let {limit, offset} = options;
+
+    if (options !== {} && options.from && options.to) {
+      if (!isValidApiRequestDateString(options.from) || !isValidApiRequestDateString(options.to)) {
+        console.error('Wrong date format! Correct format is yyyy-MM-dd');
+        return Promise.reject(new BitmovinError('Wrong date format! Correct format is yyyy-MM-dd', {}));
+      }
+      newUrl = urljoin(newUrl, options.from, options.to);
+    }
+
+    const getParams = utils.buildGetParamString({
+      limit: limit,
+      offset: offset
+    });
+
+    if (getParams.length > 0) {
+      newUrl = urljoin(newUrl, getParams);
+    }
+
+    return newUrl;
+  };
+
+  const daily = (options = {}) => {
+    let url = urljoin(configuration.apiBaseUrl, 'encoding/statistics/daily');
+    url = addOptionsToUrl(url, options);
```

Comment:
could be `const url ..` and `const urlWithOptions = addO..(url, ...)` to not reuse vars